### PR TITLE
Login page

### PR DESCRIPTION
**Pull request by Metemi**
copyied Login page from last branch

### CODE REVIEW

File: Pet-Clinic-Frontend/src/pages/Login.css [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    to {
+      transform: rotate(360deg);
+    }
+  }
+  
+  @media (prefers-reduced-motion: no-preference) {
+    a:nth-of-type(2) .logo {
+      animation: logo-spin infinite 20s linear;
+    }
+  }
```
Comment (Normelik) [inline Pet-Clinic-Frontend/src/pages/Login.css:34]: we dont need this :)

Comment (Metemi) [reply Pet-Clinic-Frontend/src/pages/Login.css]: it is gone

**Review by Normelik — COMMENTED** [review]
@Metemi css without using, it the rest of the initial project setup

**Review by Normelik — APPROVED** [review]

**Review by Normelik — APPROVED** [review]

File: Pet-Clinic-Frontend/.gitignore [hunk 2]
```diff
@@ -3,7 +3,7 @@ logs
 *.log
 npm-debug.log*
 yarn-debug.log*
-yarn-error.log*
+yarn-error.log*git
```
Comment (markkovari) [inline Pet-Clinic-Frontend/.gitignore:6]: Isn't this a duplication?

File: Pet-Clinic-Frontend/.gitignore [hunk 3]
```diff
@@ -24,4 +24,5 @@ dist-ssr
 *.sw?
 
 
-.vite/*
\ No newline at end of file
+.vite/*
+node_modules
```
Comment (markkovari) [inline Pet-Clinic-Frontend/.gitignore:28]: Already added above.

File: Pet-Clinic-Frontend/src/pages/Login.tsx [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+    <>
+      <h1>Pet Clinic</h1>
+      <h2>Login</h2>
+     <div className = "formDiv">
+        <form className='form' onSubmit={function (event){
+          event.preventDefault()
+          console.log("Form submitted")
+          console.log('username:' , username)
+          console.log('password: ' , password)
+          window.location.reload()
```
Comment (markkovari) [inline Pet-Clinic-Frontend/src/pages/Login.tsx:19]: We probably we do not want to `console.log` and reloading things with `window.reload`.

**Review by markkovari — CHANGES_REQUESTED** [review]

**Review by markkovari — APPROVED** [review]

**Pull request merged**